### ISSUE TITLE
insert size information for long reads plots

### ISSUE THREAD

**Issue by prasundutta87** [ryanlayer/samplot#160]
Hi,

I was just wondering if the insert size information in plots is important for oxford nanopore BAMs. How is it calculated if no paired end reads are present in those BAMs?

Regards,
Prasun

**Comment by jbelyeu** [comment]
Yeah good question. In the case of long reads, the axis that would convey insert size information instead conveys event size information. That is, if a read contains evidence for a deletion, duplication, or other structural variant event, it's indicated by position on the y axis where the read is placed. This is effectively the same as the change in insert distance caused by SVs, except that the default for long reads is 0 whereas for PE reads it's library prep dependent.

**Comment by prasundutta87** [comment]
So the insert size for long reads indicates the SV length?
![test_q25](https://user-images.githubusercontent.com/25484608/173647439-6f3ec10a-a7ab-466a-8df1-417d565d06c0.png)

I have some additional query for a test plot as I am trying to understand the output of an inversion event. It seems that there are multiple inversion in the region along with a deletion. I chose only inversions in my test case. Are the short stretvhes of yellow lines split reads? Also, what are the yellow dots in the bigging of the dotted blue inversion event?

**Comment by jbelyeu** [comment]
The yellow color is default for long reads, indicating a read without an event or containing a deletion, with a few events scattered around. I think the yellow dots around the ends of the blue section are the extension of the reads containing that event, but they look a little odd due to the large region size.

**Comment by prasundutta87** [comment]
Thanks, @jbelyeu. When you say "a read without an event", what do you mean? I was just wondering why will some yellow lines just appear withtout an event. The grey area shows the depth of coverage. So, shouldn't the complete area be full of yellow lines? I may be misunderstading somthing here, but please correct me if I am wrong.

**Comment by jbelyeu** [comment]
The yellow lines at the bottom are just the reads that span the region and include no SV events. These count for depth of coverage, but stack on top of each other and almost disappear from the plot. This behavior is intentional, since individual reads aren't informative unless they contain sv evidence.

**Comment by prasundutta87** [comment]
Thanks for your help with this, @jbelyeu !
